### PR TITLE
Made minor changes to the files

### PR DESCRIPTION
**Pull request by ARKEOLOGIST**
Fixed issues #28 and #29.

### CODE REVIEW

File: src/components/common/Footer.vue [hunk 1]
```diff
@@ -6,8 +6,10 @@
           v-col(cols="12" md="6").text-center
             v-layout(row)
               v-container
-                span ABOUT US
+                p ABOUT US
                 p Student Gymkhana IITJ
+                p
+                  a(href="https://devlup-labs.github.io/#/" class="white--text" style="text-decoration: none;") Devlup Labs
```
Comment (killbotXD) [inline src/components/common/Footer.vue:12]: Can you remove this link and add it to the `v-footer` and change `by Students' Gymkhana WebD Team` to `Devlup Labs`

**Review by killbotXD — CHANGES_REQUESTED** [review]
Requested a minor change

**Comment by ARKEOLOGIST** [comment]
Changes have been made as requested.

File: src/components/common/Footer.vue [hunk 2]
```diff
@@ -20,7 +21,8 @@
     v-footer(color="blue darken-2 white--text").text-center.justify-center.align-center
       h4.font-weight-regular © {{new Date().getFullYear() }} COPYRIGHT  | Made with
       v-icon(color="red").pa-1 mdi-heart
-      h4.font-weight-regular by Students' Gymkhana WebD Team
+      a(href="https://devlup-labs.github.io/#/" class="white--text" style="text-decoration: none;")
```
Comment (killbotXD) [inline src/components/common/Footer.vue:24]: can you add `target="_blank"` in here and then I think I can merge

**Comment by ARKEOLOGIST** [comment]
Done.

**Comment by killbotXD** [comment]
okay LGTM

**Review by killbotXD — APPROVED** [review]

**Pull request merged**